app/views/cookbooks/available_for_adoption.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= provide(:title, "Cookbooks available for Adoption") %>
+<%= provide(:description, "Browse and search Chef cookbooks that are available for adoption.") %>
+
+<div class="page nobg">
+<h1>Cookbooks Available for Adoption</h1>
+  <div class="heading-with-buttons">
+    <h1 class="title">
+      <%= pluralize(@number_of_available_cookbooks, 'Available Cookbook') %>
+      <small>
+        <%= link_to '<i class="fa fa-rss"></i> RSS'.html_safe, params.merge(format: 'atom'), :class => 'rss_feed_link show-for-medium-up' %>
+      </small>
+    </h1>
+  </div>
+
+  <ul class="listing">
```

Comment:
Excellent work!  I played with your code a bit to see whether I could make the cookbooks listing look just a little better for each cookbook.  I found out that we already have a style called "simple_listing" for lists and a partial called "simple_cookbook" for rendering info about a cookbook.

Would you mind replacing lines 15-22 with this content?  It just adds a bit to the presentation

```ruby
<ul class="simple_listing" %>
  <% @available_cookbooks.each do |cookbook| %>
   <li><%= render "simple_cookbook", cookbook: cookbook %></li>
 <% end %>
</ul>
```